services/motion/builtin/move_request.go

```diff
@@ -520,6 +483,7 @@ func (ms *builtIn) newMoveOnGlobeRequest(
 	mr.seedPlan = seedPlan
 	mr.replanCostFactor = valExtra.replanCostFactor
 	mr.requestType = requestTypeMoveOnGlobe
+	mr.poseOrigin = spatialmath.NewZeroPose()
```

Comment:
Why do we need to set this here?

Reply:
why not use the `startPose` we get from `newMoveRequestFromAbsolute`? wouldn't it be the zero pose for MOG?

Reply:
I think I like this idea but I'm not 100% sure it works.  What does the localizer return for MoveOnGlobe when CurrentPosition is called?  

We need this to return the absolute poses of the path in the localizers frame

Reply:
OK we talked offline I think I got it.